settings.py

```diff
@@ -96,6 +96,8 @@
 
 USE_STRATAGEM = os.getenv("USE_STRATAGEM", "false")
 
+BRANDING_PATH = os.getenv("BRANDING_PATH", "whamcloud-favicons")
```

Comment:
Can we change this to whamcloud? Eventually we should move the logos out of the binary as well.